shared-libs/contacts/test/unit/places.spec.js

```diff
@@ -561,4 +561,13 @@ describe('places controller', () => {
 
   });
 
```

Comment:
Can you please add a unit test that covers the test case I found, where you can pass an invalid contact to the new place? 
Also, can you please add some unit tests to cover the bug you are fixing?